CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,3 @@
 # Project `jupyter/docker-stacks` Code of Conduct
 
-Please see the [Project Jupyter Code of Conduct](https://github.com/jupyter/governance/blob/master/conduct/code_of_conduct.md).
+Please see the [Project Jupyter Code of Conduct](https://github.com/jupyter/governance/blob/HEAD/conduct/code_of_conduct.md).
```

Comment:
I used HEAD for links to another projects to not need to switch the links when they will decide to go from main to master.